README.md

```diff
@@ -156,6 +158,26 @@ const initialMemoryStatus = { deviceMemory: 4 };
 const { deviceMemory } = useMemoryStatus(initialMemoryStatus);
 ```
 
+### Media Capabilities
+
+`useMediaCapabilities` utility for adapting based on the user's device media capabilities
+
+```js
+import React from 'react';
+
+import { useMediaCapabilities } from 'react-adaptive-hooks/media-capabilities';
+
+const MyComponent = ({ mediaConfig }) => {
+  const { mediaCapabilities } = useMediaCapabilities(mediaConfig);
+
+  return (
+    <div>
+      { mediaCapabilities.supported ? <video muted controls>...</video> : <img src='...' /> }
```

Comment:
Am I correct in reading `mediaConfig` as being the media capabilities API config you want to be tested with `navigator.mediaCapabilities.decodingInfo()`?

e.g for the video use-case, if I wanted to test for support I could pass in something like...

```js
const mediaConfig = {
  type : 'file', // 'record', 'transmission', or 'media-source'
  video : {
    contentType : "video/webm;codecs=vp8", // valid content type
    width : 800,     // width of the video
    height : 600,    // height of the video
    bitrate : 10000, // number of bits used to encode 1s of video
    framerate : 30   // number of frames making up that 1s.
  }
}
```

to `useMediaCapabilities`. Is that right? If so, we might want to think about a more compelling usage example here. Perhaps showing an actual config being passed. The idea being that right now I think developers might not understand how this differs to some of the other utilities.

Reply:
That's exactly right, you could show a fallback image instead of playing a video but indeed that's not a good example to show what useMediaCapabilities is able to do.
I've updated the readme with a little more info and a better use case.